downloads.json

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
Would be way more useful as a normalized JSON where each item in an array has id. ie. https://github.com/paularmstrong/normalizr

Reply:
What do you need ids for? Isn't it better to access elements by name:
downloads["stable"]["Windows"]["qTox"] instead of downloads["stable"]["0"]["0"] ?

Reply:
Well it wouldn't be downloads["stable"]["0"]["0"], but my point is that it's more difficult to make a mistake with names and ids are more difficult to remember, because they don't mean anything. I don't understand why it would be more useful with ids. Maybe you could give an example?